### ISSUE TITLE
Results does not feature roles

### ISSUE THREAD

**Issue by mcspenzer** [wit-ai/wit#1258]
I would like to report a bug on two of my apps.

The current behavior is that the results do not return the associated roles for that entity. For illustration purposes, say I have the sentence:  
> _"Why am I like this?"_  

Wit should detect the _**I**_ as an entity called **_person_** with the following role of _**self**_ (person:self).
Another example would be:
> _"Why are others so cruel?_

Wit should detect the _**others**_ as an entity called **_person_** with the following role of _**others**_ (person:others)
However, the results only feature _**person**_. I am also aware that this issue should be fixed by now, but sadly I am currently experiencing it.

Here's the Application IDs of my two apps:
- _2279326185474103_
- _2828889323803130_

Thank you so much for all the help!

**Comment by blandinw** [comment]
@mcspenzer tahnsk for your feedback. This is a "bug" in your data. You have a lot of samples where the phrase "Why am I" appears, but **I** is not annotated as a **person:self**. This actively tells Wit that **I** in this case is NOT a **person:self**. Wit then tries to be smart and understand when it should and should not annotate **I** as **person:self**, but in the cases you give, Wit cannot confidently make this annotation.

TL;DR fix your dataset using the Samples tab, or the GET /samples and POST /samples APIs.
Please feel free to comment back if we can help further.

**Comment by mcspenzer** [comment]
Alright I see my mistake now. Actively retraining the bots to include the said roles. Thanks!
